src/Sulu/Component/Content/Tests/Functional/SmartContent/SmartContentQueryBuilderTest.php

```diff
@@ -784,10 +787,10 @@ public function testOrderBy()
         );
         $result = $this->contentQuery->execute('sulu_io', ['en'], $builder);
 
-        $this->assertEquals('QWERTZ', $result[0]['title']);
-        $this->assertEquals('qwertz', $result[1]['title']);
-        $this->assertEquals('ASDF', $result[2]['title']);
-        $this->assertEquals('asdf', $result[3]['title']);
```

Comment:
Looks like I've made some changes that broke this and the next test (although I can't find out which ones...)

But after looking at this test I came to the conclusion that these tests are unpredictable anyway. The title will be lowered for the query, so you can't really say if `asdf` or `ASDF` will come first. It's also strange here that the result using `desc` is not exactly reversed from the other one.

So I changed the test to see the not all capital letters come first or vice versa, i.e. that the ordering is case insensitive.